frontend/src/pages/projects/screens/detail/connections/ConnectionsList.tsx

```diff
@@ -22,7 +22,8 @@ const ConnectionsList: React.FC = () => {
     connections: { data: connections, loaded, error, refresh: refreshConnections },
     currentProject,
   } = React.useContext(ProjectDetailsContext);
-  const [connectionTypes, connectionTypesLoaded, connectionTypesError] = useWatchConnectionTypes();
+  const [connectionTypes, connectionTypesLoaded, connectionTypesError] =
+    useWatchConnectionTypes(true);
```

Comment:
I thought out this when I made the change. However I'm ok changing this back so that we can edit connections who's types have since been disabled.

Reply:
Is the final verdict to change it back? if so it's still failing the lookup from [line 179](https://github.com/opendatahub-io/odh-dashboard/pull/3326/files#diff-30f2cd16b3dc39b2c29a608879568973b39dddf1588ba2eac82e1e573c9003feR179) 
```
options={!isEdit ? enabledConnectionTypes : undefined}
```
![image](https://github.com/user-attachments/assets/87452ea2-d173-494f-8a1f-45a4da91cd4d)

Reply:
@emilys314 this is working for me as well as one of the unit tests. 
I have an `s3` connection type which is disabled. I then create a data connection and then edit it from the connections tab. The dropdown is disabled and shows the correct type in the dropdown. Similarly it in the table it also shows the correct type name.

As for line 179. When editing, I made it so that we do not supply options which will result in a disabled dropdown and it will create the single option based on the connection type passed it for editing.

Reply:
Oh okay sorry, looks like my cluster didn't update or something, it's working now